components/HelplineCard.jsx

```diff
@@ -69,7 +70,7 @@ const HelplineCard = ({
       <div className="flex justify-between items-center mx-2 mt-2 pb-3">
         <div className="font-semibold">{description}</div>
         <div className="font-mono text-gray-700 text-sm">
-          {parseDateString(createdTime)}
+          Verified @ {parseDateString(lastVerifiedOn)}
```

Comment:
Hey Have a [Look at this](https://life-git-fork-karmakoder-karmakoder-coronasafe.vercel.app/uttar_pradesh/allahabad/helpline)

<img width="246" alt="Screenshot 2021-04-23 at 9 01 33 PM" src="https://user-images.githubusercontent.com/61158210/115894613-17a9ce80-a477-11eb-9b57-e819ff5832b7.png">

Would be good if the value exists and then showing the `lastVerified`.